docs/source/models.rst

```diff
@@ -1304,129 +1304,105 @@ For more info, see:
+    # Write the deployment configuration into a file.
+    deployment_config_path = "deployment_config.json"
+    with open(deployment_config_path, "w") as outfile:
+        outfile.write(json.dumps(deploy_config))
 
-.. code-block:: bash
+    # Set the tracking uri as the deployment client.
+    client = get_deploy_client(mlflow.get_tracking_uri())
 
-    # note mlflow azureml build-image is being deprecated, it will be replaced with a new command for model deployment soon
-    mlflow azureml build-image -w <workspace-name> -m <model-path> -d "Wine regression model 1"
+    # MLflow requires the deployment configuration to be passed as a dictionary.
+    config = {'deploy-config-file': deployment_config_path}
+    model_name = "mymodel"
+    model_version = 1
 
-    az ml service create aci -n <deployment-name> --image-id <image-name>:<image-version>
+    # define the model path and the name is the service name
+    # if model is not registered, it gets registered automatically and a name is autogenerated using the "name" parameter below 
+    client.create_deployment(model_uri=f'models:/{model_name}/{model_version}',
+                            config=config,
+                            name="mymodel-aci-deployment")
+
+    # After the model deployment completes, requests can be posted via HTTP to the new ACI
+    # webservice's scoring URI. 
+    print("Scoring URI is: %s", webservice.scoring_uri)
+
+.. rubric:: Example: Workflow using the MLflow CLI
+
+.. code-block:: bash
+
+    mlflow deployments create --name <deployment-name> -m models:/<model-name>/<model-version> -t <azureml-mlflow-tracking-url> --deploy-config-file deployment_config.json
 
     # After the image deployment completes, requests can be posted via HTTP to the new ACI
-    # webservice's scoring URI. The following example posts a sample input from the wine dataset
-    # used in the MLflow ElasticNet example:
-    # https://github.com/mlflow/mlflow/tree/master/examples/sklearn_elasticnet_wine
+    # webservice's scoring URI.
 
     scoring_uri=$(az ml service show --name <deployment-name> -v | jq -r ".scoringUri")
 
-    # `sample_input` is a JSON-serialized pandas DataFrame with the `split` orientation
-    sample_input='
-    {
-        "columns": [
-            "alcohol",
-            "chlorides",
-            "citric acid",
-            "density",
-            "fixed acidity",
-            "free sulfur dioxide",
-            "pH",
-            "residual sugar",
-            "sulphates",
-            "total sulfur dioxide",
-            "volatile acidity"
-        ],
-        "data": [
-            [8.8, 0.045, 0.36, 1.001, 7, 45, 3, 20.7, 0.45, 170, 0.27]
-        ]
-    }'
+You can also test your deployments locally first using the option `run-local`:
+
+.. code-block:: bash
 
-    echo $sample_input | curl -s -X POST $scoring_uri\
-    -H 'Cache-Control: no-cache'\
-    -H 'Content-Type: application/json'\
-    -d @-
```

Comment:
We seem to be removing some valuable example content. Can we keep it around?

Reply:
I removed cause the requests are exactly the same as any request accepted by MLFlow serve, so this is no specific to Azure ML. I'd rather point users to that documentation. Does it make sense or do you think we should include it here too?

Reply:
I think there's something to be said for having the example in place here. Let's add that back in so as to not have users have to context switch to a different location while reading through how to do this.

Reply:
Sure. I added them back on [73a18e6](https://github.com/mlflow/mlflow/pull/5491/commits/73a18e6198992ff818df2d0564632662cd2f169d).